app/src/main/java/org/ole/planet/myplanet/ui/dashboard/BellDashboardFragment.java

```diff
@@ -60,23 +54,23 @@ private void declareElements(View view) {
 //            startActivity(new Intent(getActivity(), SettingActivity.class));
 //        });
         initView(view);
-       // view.findViewById(R.id.ll_achievement).setOnClickListener(v -> homeItemClickListener.openCallFragment(new AchievementFragment()));
-       // view.findViewById(R.id.ll_news).setOnClickListener(v -> homeItemClickListener.openCallFragment(new NewsFragment()));
+        // view.findViewById(R.id.ll_achievement).setOnClickListener(v -> homeItemClickListener.openCallFragment(new AchievementFragment()));
```

Comment:
@leomaxi i think we should remove this comment, it is not required anymore and  i don't like comments in the code :D